diff --git a/typo3/sysext/rsaauth/Classes/Backend/CommandLineBackend.php b/typo3/sysext/rsaauth/Classes/Backend/CommandLineBackend.php
index 74d58e2..0c637df 100644
--- a/typo3/sysext/rsaauth/Classes/Backend/CommandLineBackend.php
+++ b/typo3/sysext/rsaauth/Classes/Backend/CommandLineBackend.php
@@ -60,6 +60,20 @@ class CommandLineBackend extends \TYPO3\CMS\Rsaauth\Backend\AbstractBackend {
 	}
 
 	/**
+	 * Denies deserialization.
+	*/
+	public function __wakeup()
+	{
+		$this->opensslPath = null;
+		$this->temporaryDirectory = null;
+		
+		throw new \RuntimeException(
+			__CLASS__ . ' cannot be unserialized',
+			1531336156
+		);
+	}
+
+	/**
 	 * Creates a new key pair for the encryption or gets the existing key pair (if one already has been generated).
 	 *
 	 * There should only be one key pair per request because the second private key would overwrites the first private
@@ -154,4 +168,4 @@ class CommandLineBackend extends \TYPO3\CMS\Rsaauth\Backend\AbstractBackend {
 		return $result;
 	}
 
-}
\ No newline at end of file
+}
