diff --git a/t3lib/config_default.php b/t3lib/config_default.php
index 7044a6a..5f76424 100644
--- a/t3lib/config_default.php
+++ b/t3lib/config_default.php
@@ -15,10 +15,10 @@
 if (!defined ('PATH_typo3conf')) 	die ('The configuration path was not properly defined!');
 
 //Security related constant: Default value of fileDenyPattern
-define('FILE_DENY_PATTERN_DEFAULT', '\.(php[3-6]?|phpsh|phtml)(\..*)?$|^\.htaccess$');
+define('FILE_DENY_PATTERN_DEFAULT', '\.(php[3-6]?|phpsh|phtml|pht)(\..*)?$|^\.htaccess$');
 
 //Security related constant: Comma separated list of file extensions that should be registered as php script file extensions
-define('PHP_EXTENSIONS_DEFAULT', 'php,php3,php4,php5,php6,phpsh,inc,phtml');
+define('PHP_EXTENSIONS_DEFAULT', 'php,php3,php4,php5,php6,phpsh,inc,phtml,pht');
 
 // Defines a list that are basically required by a TYPO3 system.
 define('REQUIRED_EXTENSIONS', 'cms,lang,sv,em,recordlist');
