diff --git a/t3lib/config_default.php b/t3lib/config_default.php
index 7faaa40..bdc0a44 100644
--- a/t3lib/config_default.php
+++ b/t3lib/config_default.php
@@ -17,10 +17,10 @@
 if (!defined ('PATH_typo3conf')) 	die ('The configuration path was not properly defined!');
 
 //Security related constant: Default value of fileDenyPattern
-define('FILE_DENY_PATTERN_DEFAULT', '\.(php[3-6]?|phpsh|phtml)(\..*)?$|^\.htaccess$');
+define('FILE_DENY_PATTERN_DEFAULT', '\.(php[3-6]?|phpsh|phtml|pht)(\..*)?$|^\.htaccess$');
 
 //Security related constant: Comma separated list of file extensions that should be registered as php script file extensions
-define('PHP_EXTENSIONS_DEFAULT', 'php,php3,php4,php5,php6,phpsh,inc,phtml');
+define('PHP_EXTENSIONS_DEFAULT', 'php,php3,php4,php5,php6,phpsh,inc,phtml,pht');
 
 $TYPO3_CONF_VARS = array(
 	'GFX' => array(		// Configuration of the image processing features in TYPO3. 'IM' and 'GD' are short for ImageMagick and  GD library respectively.
@@ -377,8 +377,8 @@ $T3_VAR = array();	// Initialize.
 $TYPO_VERSION = '4.4.16-dev';	// deprecated: use the constants defined below
 define('TYPO3_version', $TYPO_VERSION);
 define('TYPO3_branch', '4.4');
-define('TYPO3_copyright_year', '1998-2012');
-define('TYPO3_user_agent', 'User-Agent: TYPO3/'.TYPO3_version);
+define('TYPO3_copyright_year', '1998-' . date('Y'));
+define('TYPO3_user_agent', 'User-Agent: TYPO3');
 
 // Database-variables are cleared!
 $typo_db = '';					// The database name
@@ -615,4 +615,4 @@ $SIM_EXEC_TIME = $EXEC_TIME;			// $SIM_EXEC_TIME is set to $EXEC_TIME but can be
 $ACCESS_TIME = $EXEC_TIME - ($EXEC_TIME % 60);		// $ACCESS_TIME is a common time in minutes for access control
 $SIM_ACCESS_TIME = $ACCESS_TIME;		// if $SIM_EXEC_TIME is changed this value must be set accordingly
 
-?>
\ No newline at end of file
+?>
